### ISSUE TITLE
app(web): Auto-expand Web Search Provider URL to allow Newlines for multi-search

### ISSUE THREAD

**Issue by GregSweats** [adirh3/Fluent-Search#975]
# Issue

**Is your feature request related to a problem? Please describe.**
No, it is a request.

**Describe the solution you'd like**
Found out about MULTI-SEARCH using spaces to separate multiple "Web search tags".
(engines? MS Edge term for custom Searches in Settings, make M$ boi feel right at home ❤️).

Super awesome feature, but hidden in the description, and hard to actually enter multiple search engines.

I wanted to search across three of my websites. For each website I:

1. Went to Website
2. Performed a test search (`asdf23`)
3. Copied the Website URL from browser's Address Bar
4. Opened Notepad (`[WIN]+[R], notepad, [ENTER]`)
5. Pasted URL
6. Replaced placeholder (`asdf23`) with `%s`
7. Go to next website and repeat.

Then, I can paste that newline separated list into `VS Code` to use multi-carat editing:

1. Paste newline separated list of Search Engine URLs
2. Select each line (`[CTRL+A]`)
3. Turn into multi-carat at start of each line (`[CTRL+SHIFT+I]`)
4. Go to start of new line (`[HOME]`)
5. `[BACKSPACE]`
6. `[SPACE]`
7. `[CTRL+A]`
8. `[CTRL+C]`

Now to enter into Fluent...

1. Back to Fluent Settings
2. Paste in space-separated list of URLs.

Now I knew it felt like a _little_ work but typing out the steps is crazy.
Will refine this Issue more later, but that is general gist of it.

I would like the `Settings -> Apps -> Web`

**Describe alternatives you've considered**
Dealing with space separated options and pasting URL into Fluent.

#### (new-line separated, like a human would do)

```text
https://www.example8.com/searches.html?q=%s
https://www.exampletube.com/search?q=%s
https://www.youexample.com/search/?query=%s
```

### Space Separated

Current Fluent Search way, like a developer trying to produce an awesome feature, and did!

#### Screenshot 1. Settings

![Screenshot 1. Settings](https://github.com/adirh3/Fluent-Search/assets/106463642/8969aa39-a13c-461c-bf0c-664d12691156)

#### Screenshot 2. Gross URLs when line-wrapped

![Screenshot 2. Gross URLs when line-wrapped](https://github.com/adirh3/Fluent-Search/assets/106463642/6ecf9d03-e13f-483f-a2fc-00ec5eddc53a)

**Comment by GregSweats** [comment]
Markdown was ran past a linter, required a first-level heading at top of page.

**Comment by adirh3** [comment]
This issue has been potentially fixed in version 1.0.0.33                                          . Changelog - https://install.appcenter.ms/users/adirh3-gmail.com/apps/fluent-search-alpha/distribution_groups/exe

**Comment by GregSweats** [comment]
Looks like this one:

> Changed Web search multi URL feature to separate URLs based on new line (instead of space) [#975](https://github.com/adirh3/Fluent-Search/issues/975)

Also excited for native Firefox operations:

> Added support searching in FireFox profiles in Browser search, you can remove the FireFox plugin [#924](https://github.com/adirh3/Fluent-Search/issues/924)

_p.s. Nothing you did wrong, just my stylistic (grammar police) eyes, but it is "Firefox". I just double checked. Hoping to dig in here soon, find the translation files and make a few tweaks, just suggestions, should be a good PR. Might be a few weeks though while wrapping-up some hectict-ness after a delayed launch at work._

Do you happen to have a "verify-after-release" label, or something similar to keep track of tickets like this and #975 where you say "potentially"? Probably no biggie, and maybe many people don't reply after filing a bug or feature request, and simply see it in the next release. Just trying to learn your flow.

Thanks Adir!
